iml-agent/src/cli.rs

```diff
@@ -489,7 +515,27 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         }
         App::GetKernel { modules } => match check_kernel::get_kernel(modules).await {
             Ok(s) => println!("{}", s),
-            Err(e) => println!("{:?}", e),
+            Err(e) => {
+                eprintln!("{:?}", e);
+                exit(exitcode::SOFTWARE);
+            }
+        },
+        App::IsMounted { mount_point } => match lustre::is_mounted(mount_point).await {
+            Ok(x) => println!("{}", x),
+            Err(e) => {
+                eprintln!("{:?}", e);
+                exit(exitcode::SOFTWARE);
+            }
+        },
+        App::TryMount {
+            lustre_device,
+            mount_point,
+        } => match lustre::try_mount((lustre_device, mount_point)).await {
+            Ok(()) => println!("Done"),
```

Comment:
Since `try_mount` returns unit `()`, so I'm printing "Done" just to print something...